src/components/clientDashboard/enrollments/EnrollmentReminders.tsx

```diff
@@ -0,0 +1,117 @@
+import { Box } from '@mui/material';
+
+import Loading from '@/components/elements/Loading';
+import GenericTable from '@/components/elements/table/GenericTable';
+import { ColumnDef } from '@/components/elements/table/types';
+import TitleCard from '@/components/elements/TitleCard';
+import { parseAndFormatDate } from '@/modules/hmis/hmisUtil';
+import { ClientDashboardRoutes } from '@/routes/routes';
+
+import {
+  AssessmentRole,
+  ReminderFieldsFragment,
+  ReminderTopic,
+  useGetEnrollmentRemindersQuery,
+} from '@/types/gqlTypes';
+import generateSafePath from '@/utils/generateSafePath';
+
+const describeReminder = (reminder: ReminderFieldsFragment): string => {
+  switch (reminder.topic) {
+    case ReminderTopic.AnnualAssessment:
+      return 'Annual Assessment';
+    case ReminderTopic.AgedIntoAdulthood:
+      return 'Update Assessment';
+    case ReminderTopic.IntakeIncomplete:
+      return 'Intake Assessment';
+    case ReminderTopic.ExitIncomplete:
+      return 'Exit Assessment';
```

Comment:
Maybe changing the wording here, since this one doesn't (shouldn't) have a due date / overdue status
```suggestion
      return 'Incomplete Exit';
```